karapace/kafka_rest_apis/consumer_manager.py

```diff
@@ -208,21 +196,20 @@ async def create_consumer(self, group_name: str, request_data: dict, content_typ
             KarapaceBase.r(content_type=content_type, body={"base_uri": base_uri, "instance_id": consumer_name})
 
     async def create_kafka_consumer(self, fetch_min_bytes, group_name, internal_name, request_data):
+        ssl_context = create_client_ssl_context(self.config)
         while True:
             try:
                 session_timeout_ms = self.config["session_timeout_ms"]
                 request_timeout_ms = max(
                     session_timeout_ms,
-                    KafkaConsumer.DEFAULT_CONFIG["request_timeout_ms"],
+                    305000,  # kafka-python's DEFAULT_CONFIG["request_timeout_ms"]
```

Comment:
This copies kafka-python's default, which the value there has been because aiokafka doesn't have this exposed, and I didn't want to import it from kafka-python here... I didn't get if you had any better suggestion for this.